emoji-copy@felipeftn/emojiButton.js

```diff
@@ -96,7 +96,7 @@ export class EmojiButton {
       symbol == Clutter.KP_Enter
     ) {
       let emojiToCopy = this.getTaggedEmoji();
-      let [mods] = global.get_pointer();
+      let [x, y, mods] = global.get_pointer();
```

Comment:
Just a quick add here, could you please add `_` before the `x` and `y` variables?
Linter may get upset if not :sweat_smile: 
```suggestion
      let [_x, _y, mods] = global.get_pointer();
```